src/App.tsx

```diff
@@ -21,14 +24,20 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              { todos.length ? (<TodoList />) : (<Loader />)}
             </div>
           </div>
         </div>
       </div>
-
-      <TodoModal />
+      { selectedTodo && (<TodoModal />)}
     </>
   );
 };
+
+export const App: React.FC = () => {
+  return (
```

Comment:
there is no need to write word `return` if you don'y have any logic, just jsx

rewrite to
`= () => (`